lib/UnoCore/Targets/Android/app/src/main/AndroidManifest.xml

```diff
@@ -33,7 +33,10 @@
                   #endif
                   />
     #endif
-
+    
+#if @(Project.Android.SupportsAndroidGo:IsSet)
```

Comment:
I think this line should be the following, so that the value of the property is respected. Otherwise it looks like we'll insert the same block whether the value is `true` or `false`.

```suggestion
#if @(Project.Android.SupportsAndroidGo:Test(1, 0))
```